.github/workflows/main.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           name: src
       - uses: actions/setup-go@v2
         with:
-          go-version: "1.13"
+          go-version: "1.17"
```

Comment:
Is there a specific reason for running the tests with Go 1.17? To me it doesn't seem like a bad idea to keep supporting older versions, assuming that doesn't take a lot of effort?

Reply:
Indeed, there is no reason to test with a later version.